### PR TITLE
windows: update link to docker engine installation

### PR DESCRIPTION
**Pull request by mweibel**
the existing link unfortunately is a dead link and doesn't redirect to the new place properly. Linking directly probably makes sense.

### CODE REVIEW

**Comment by mweibel** [comment]
CLA should be signed now. Sorry only completed half of the steps before opening the PR ;)

**Comment by neolit123** [comment]
thanks for signing the CLA; we had a similar PR here:
https://github.com/kubernetes/website/pull/21862

/lgtm
/sig windows cluster-lifecycle

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/dow
 ### Joining a Windows worker node
 {{< note >}}
 You must install the `Containers` feature and install Docker. Instructions
-to do so are available at [Install Docker Engine - Enterprise on Windows Servers](https://docs.docker.com/ee/docker-ee/windows/docker-ee/#install-docker-engine---enterprise).
+to do so are available at [Install Docker Engine - Enterprise on Windows Servers](https://docs.mirantis.com/docker-enterprise/v3.1/dockeree-products/docker-engine-enterprise/dee-windows.html).
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:143]: I think the link should be to https://docs.docker.com/desktop/enterprise/admin/install/windows/

**Pull request merged**